packages/nodes-base/nodes/WhatsApp/GenericFunctions.ts

```diff
@@ -102,3 +104,27 @@ export async function appWebhookSubscriptionDelete(
 		payload: { object },
 	});
 }
+
+export const createMessage = (
```

Comment:
I meant extracting only the construction of the body part of the request, but this is also fine 👍 Message is quite generic term. Could we make this more specific? Would something like `createConfirmationMessageOptions` work?